doajtest/testbook/dashboard/historical_stats.yml

```diff
@@ -0,0 +1,97 @@
+suite: Dashboard
+testset: Historical Statistics
+tests:
+  - title: Historical Statistics Display - Managing Editor with 3 groups
+    context:
+      role: managing editor/admin
+      testdrive: statistics
+    steps:
+      - step: Navigate to "/testdrive/statistics" and wait for the page to load slowly. This should be done once at the beginning of the test session and not refreshed between individual tests.
+      - step: Log in with provided managing_editor credentials.
+        results:
+          - User dashboard is displayed.
+      - step: Verify the motivational banner at the top of the dashboard (green) and check number of finished applications.
+        results:
+          - The motivational banner is GREEN and shows the correct sum of applications finished in all groups.
+      - step: Verify historical statistics in the Activity section at the bottom.
+        results:
+          - Three groups are displayed.
+          - Only associate editors are shown in the Activity section for each group.
+          - Each group has the correct statistics for the current year.
+          - The number of READY/COMPLETED applications matches the "additional information" table.
+      - step: Log out.
+      - step: (Optional) Click the link at the bottom of the testdrive page to free resources after finishing this test session.
+  - title: Historical Statistics Display - Editor with 2 groups
+    context:
+      role: editor_0
+      testdrive: statistics
+    steps:
+      - step: Navigate to "/testdrive/statistics" and wait for the page to load slowly. This should be done once at the beginning of the test session and not refreshed between individual tests.
+      - step: Log in with provided editor_0 credentials.
+        results:
+          - User dashboard is displayed.
+      - step: Verify the motivational banner at the top of the dashboard (green) and check number of finished applications.
+        results:
+          - The motivational banner is GREEN and shows the correct sum of applications finished in both groups.
+      - step: Verify historical statistics in the Activity section at the bottom.
+        results:
+          - Two groups are displayed.
+          - Only associate editors are shown in the Activity section for each group.
+          - Each group has the correct statistics for the current year.
+          - The number of READY/COMPLETED applications matches the "additional information" table.
```

Comment:
When I do this test, I find that the groups names in the tabs all have empty brackets by them.  This also happens on my own account, not using the testdrive.  Something up with the renderer there?

Reply:
![Screenshot from 2024-09-05 14-21-57](https://github.com/user-attachments/assets/fd214bbe-2d43-4f3a-a1d9-7f51a52153cf)

Reply:
Yep, it doesn't display 0 for groups with no applications. Fixed.

Reply:
@richard-jones  
It currently is actually without the '0'. And the toolbar is displayed on the tab with the text `"Toggle to {{ eg.name }}’s activity data"`. Do we want to change that with that PR? Adding a separate toolbar just to the number can be a bit weird.

I think we could rethink how to display the tabs with 0 applications, but I would leave it for another PR.

![image](https://github.com/user-attachments/assets/a94c3a00-9687-49d6-b139-03db0f4efa9d)